kahuna/package.json

```diff
@@ -49,6 +50,16 @@
             ]
           }
         }
+      },
+      "github:quantumui/titip@397fd324d5b5cc98ee332f876a100a70f69b273e": {
```

Comment:
Forked it to my account as I don't have perms for organisation account. I've also [removed](https://github.com/quantumui/titip/pull/2) the default cursor styling.